app/azure_logging.py

```diff
@@ -45,22 +72,20 @@ def initialize_logging(environment: str, logging_level: int, correlation_id: Opt
         azurelog_handler.addFilter(ExceptionTracebackFilter())
         logger.addHandler(azurelog_handler)
     except ValueError as e:
-        if environment == "local":
-            logger.info(f"Application Insights logger handler not set.: {e}") 
+        if os.environ.get("APPLICATIONINSIGHTS_CONNECTION_STRING") is None:
+            logger.info(f"Application Insights logger handler not set")
```

Comment:
For what it is worth this is not visible to the user, the first log that works is `Logging initialised.`

Reply:
Scratch that, works perfectly!